eternalcore/src/test/java/com/eternalcode/core/util/RandomUtilTest.java

```diff
@@ -0,0 +1,35 @@
+package com.eternalcode.core.util;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import panda.std.Option;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RandomUtilTest {
+
+    @Test
+    void testRandomElement() {
+        Collection<String> strings = Arrays.asList("a", "b", "c", "d", "e");
+        Option<String> randomElement = RandomUtil.randomElement(strings);
+
+        assertNotNull(randomElement);
```

Comment:
to nie ma sensu `randomElement` jest Option i nigdy nie jest null sam w sobie
sprawdź Option#isPresent()